src/cape_ros_node.cpp

```diff
@@ -219,19 +241,33 @@ void CapeRosNode::depthCallback(const sensor_msgs::ImagePtr& image)
       }
       else
       {
-        ROS_ERROR("Failed to obtain camera intrinsic messages");
+        ROS_ERROR("Cape: Failed to obtain camera intrinsic messages");
         return;
       }
-      ROS_WARN("fx, fy, px, py: %f, %f, %f, %f", fx_, fy_, cx_, cy_);
+      ROS_WARN("Cape: fx, fy, px, py: %f, %f, %f, %f", fx_, fy_, cx_, cy_);
     }
     catch (std::exception e)
     {
-      ROS_ERROR("Failed to obtain camera intrinsic messages: %s", e.what());
+      ROS_ERROR("Cape: Failed to obtain camera intrinsic messages: %s", e.what());
       return;
     }
   }
 
-  cv_bridge::CvImageConstPtr image_ptr = cv_bridge::toCvShare(image, image->encoding);
+  cv_bridge::CvImageConstPtr image_ptr;
+  try
+  {
+    image_ptr = cv_bridge::toCvShare(image, image->encoding);
+  }
+  catch (const cv_bridge::Exception& e)
+  {
+    ROS_ERROR_STREAM("Could not convert depth image. " <<
```

Comment:
I'm noticing that some ROS_ERROR messages start with "Cape:" while others don't. I'm assuming they should all begin the same way for consistency.